docs/admin-guide/server.md

```diff
@@ -91,4 +106,15 @@ gunicorn \
     scout.server.auto:app
 ```
 
+If you are running a larger environment, where this is one component, we encourage a reverse proxy configuration where Scout is served by gunicorn, and reverse proxied by NGIX. Then NGIX will handle the secure communication, and the start config reduces to something like
+```bash
+SCOUT_CONFIG="/full/path/to/scout_flask_conf.py" \
+gunicorn \
+    --workers=4 \
+    --bind="HOST:PORT" \
+    --access-logfile=-
+    --error-logfile=-
+    wsgi_gunicorn:app
```

Comment:
```suggestion
    scout.server.auto:app
```

Reply:
Hm, but we do recast it as wsgi in the app (you know, the proxy fix stuff)? We can test: my guess is both will actually work.

Reply:
Yes but the question from the user was about this, and the fact that the file was not found..

Reply:
Right you are, ofcourse! We should also remove the corresponding from the servers repo..